tests/console/docker_image.pm

```diff
@@ -58,30 +58,32 @@ sub run {
         chmod +x container-diff-linux-amd64 && sudo mv container-diff-linux-amd64 /usr/local/bin/container-diff"
     );
 
-    foreach my $image_name (@image_names) {
+    for my $i (0 .. $#image_names) {
```

Comment:
Related to comment above - this cycle is always `for my $i (0 .. 0) {..}`. Why not remove it?